client/db/bolt/db_test.go

```diff
@@ -533,6 +534,38 @@ func TestOrders(t *testing.T) {
 	if err == nil {
 		t.Fatalf("no error encountered for updating unknown order change coin")
 	}
+
+	numToDo = 5
+	numActiveDex := numToDo
+	acct := acct1
+	base, quote := uint32(42), uint32(0) // base - dcr, quote - btc.
+	nTimes(numToDo, func(i int) {
+		status := order.OrderStatusBooked
+		ord := randOrderForMarket(base, quote)
```

Comment:
As unlikely as it is, couldn't this potentially result in `ActiveDexMarketOrders` returning more orders than expected because it catches an active order from earlier in the test? Probably won't matter once you start using the order from `(*dexConnection).trades`, in which case `ActiveDexMarketOrders` is no longer needed.

Reply:
Yup, `ActiveDexMarketOrders` will no longer be needed if the `metaOrders` are sourced from `(*dexConnection).trades`. Test will be removed.